pkg/client_data/lib/package_api.dart

```diff
@@ -154,11 +154,15 @@ class VersionInfo {
 class VersionScore {
   final int grantedPoints;
   final int maxPoints;
+  final int likeCount;
```

Comment:
Can we rename this to `likes`

Reply:
I'd argue that we should use `likeCount` as opposed to `likes` not only here, but other places where we've had `likes` before. `List<Like>` should be `likes` (and `likes` should be `List<Like>`), while the cardinality of the same list is the `likeCount`.

Reply:
Seems fine either way, I just wanted to try to keep the external API neat.